docs/main/Solutioning/backup-and-restore/kasten-br-fs.md

```diff
@@ -0,0 +1,338 @@
+---
+id: kastenbrfs
+title: Kasten Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
+keywords: 
+  - Kasten Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
+  - Kasten Backup and Restore
+  - FileSystem
+description: In this document, you learn about Kasten Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem.
+---
+
+# Kasten Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
+
+Using Kasten K10 for backup and restore operations with Replicated PV Mayastor snapshots combines the strengths of both tools, providing a robust, high-performance solution for protecting your Kubernetes applications and data. This integration ensures that your stateful applications are protected, with the flexibility to recover quickly from any failure or to migrate data as needed, ensuring your data is always protected and recoverable.
+
+In this guide, we will utilize Kasten to create a backup of a sample Nginx application with a Replicated PV Mayastor from a cluster, transfer the backup to an object store, and restore it on a different cluster.
+
+## Requirements
+
+### Replicated PV Mayastor
+
+Replicated PV Mayastor, a high-performance, container-native storage that provides persistent storage for Kubernetes applications. It supports various storage types, including local disks, NVMe, and more. It integrates with Kubernetes and provides fast, efficient snapshots and creates snapshots at the storage layer, providing a point-in-time copy of the data. These snapshots are highly efficient and consume minimal space, as they only capture changes since the last snapshot.
+
+Make sure that Replicated PV Mayastor has been installed, pools have been configured, and applications have been deployed before proceeding to the next step. Refer to the [OpenEBS Installation Documentation](../../quickstart-guide/installation.md#installation-via-helm) for more details.
```

Comment:
This is a generic statement.
Since we are planning to backup a test nginx application, we need to provide the yaml to install the application. Also, it's important that the volume mode is specified as 'filesystem'.

Reply:
@balaharish7 YAML added now to confluence, please update the doc

Reply:
Yaml added to the docs